ArcGISRuntimeSDKQt_QMLSamples/Layers/CreateAndSaveKmlFile/CreateAndSaveKmlFile.qml

```diff
@@ -57,6 +57,10 @@ Rectangle {
                 if (saveStatus === Enums.TaskStatusErrored) {
                     console.log(`Error: ${error.message} - ${error.additionalMessage}`);
                 }
+
+                if (saveStatus === Enums.TaskStatusCompleted) {
```

Comment:
This can be an `else if` statement. These are mutually exclusive states.

Or, the `Enums.TaskStatusErrored` case can return.